lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -41,9 +41,9 @@ def self.commit_version_bump(include_deliverfile: true, include_metadata: true)
         def self.localize_project
           Action.sh("cd #{ENV['PROJECT_ROOT_FOLDER']} && ./Scripts/localize.py")
 
-          strings_files = Dir.chdir(File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'])) do
-            Dir.glob('*.lproj/*.strings')
-          end
+          localizations_path = File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'])
+          strings_files = Dir.glob(File.join(localizations_path, '*.lproj/*.strings'))
```

Comment:
That being said, as pointed out in the PR review, in WC and WP those are not the path to the `.lproj` files either, so that won't probably fix it for them.

Reply:
> You started using File.join for first part, but didn't use it for the `*.lproj/*.strings` part

🙃

Reply:
> Note: slight risk with this solution is we open up to wildcard injection: if either of the PROJECT_ROOT_FOLDER or PROJECT_NAME env var contains a wildcard glob character (* or ? typically), this part of the path too will end up being interpreted by Dir.glob. That's very likely why this was done via a Dir.chrdir before.

IMHO this is pretty unlikely, so I'd say just keep it clean for now 🤷‍♂️